packages/relayer/src/services/relay.ts

```diff
@@ -87,12 +77,44 @@ export const relayer = async (
     };
   }
 
+  if (reciver_storage) {
+    await account.functionCall({
+      contractId: tokenId,
+      methodName: "storage_deposit",
+      args: {
+        account_id: reciver_storage,
+        registration_only: true,
+      },
+      gas: AttachedGas as any,
+      attachedDeposit: "2350000000000000000000" as any,
+    });
+  }
+
+  const relayerStorage = await getTokenStorage(
+    tokenId,
+    env.ACCOUNT_ID,
+    env.RPC_URL
+  );
+
+  if (!relayerStorage || relayerStorage.total < "0.10") {
```

Comment:
This is not how storage for NEP-141 contracts work.

In more complex contracts, where the storage consumption of a user varies according to the use of the contracts this setup would make sense (as was the casse in JUMP DeFi).

In token contracts, however, storage is fixed. After you register you never have to pay storage again. Therefore it is only necessary to evaluate `!relayerStorage`, there is no need to look at `relayerStorage.total`.